addons/website_forum/views/website_forum.xml

```diff
@@ -1618,6 +1618,7 @@
                         <div class="tab-content" id="o_tab_content_spam">
                             <div class="tab-pane fade show active" data-key="create_uid" id="spam_user" role="tabpanel" aria-labelledby="user-tab">
                                 <form class="row" >
+                                    <t t-set="env_user" t-value="user"/>
```

Comment:
While changing the TYPO and commit message lines length, this could deserve a comment in the HTML, as it's far from straightforward to figure for the next person jumping into this code later.
Something like
```xml
<!-- Prevent the foreach loop to overide the `user` variable given by the controller -->
```
(mind the 80 char max per line for comments)

Reply:
@rdeodoo updated the typo and added the comment in the code